### PR TITLE
Update us.m3u

### LINKED ISSUE #3180: Add: Telemundo
**Channel Name:** Telemundo
**Country:** USA
**Language:** xxx
**External Link (optional):** 
**Notes (optional):** Located in Phoenix, AZ - https://en.wikipedia.org/wiki/KTAZ

### PR DESCRIPTION
**Pull request by Sphinxroot**
close #3180

### CODE REVIEW

**Comment by Dum4G** [comment]
We have decided to include only adaptive links from now on. Only the master one is needed

**Comment by Sphinxroot** [comment]
oupss me bad ive forgot  about it   now fixed

**Comment by LaneSh4d0w** [comment]
Added call signs and fixed.

**Review by BellezaEmporium — APPROVED** [review]

File: channels/us.m3u [hunk 1]
```diff
@@ -3304,9 +3304,9 @@ https://bcoveliveios-i.akamaihd.net/hls/live/206632/1997976452001/TCTHLS/playlis
 https://bcoveliveios-i.akamaihd.net/hls/live/206632/1997976452001/FamilyHLS/playlist.m3u8
 #EXTINF:-1 tvg-id="TechnoWarehouseUS.us" tvg-name="Techno Warehouse (US)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/GyxXMDv.png" group-title="Music",Techno Warehouse (US) (1080p)
 https://eu-nl-012.worldcast.tv/dancetelevisionthree/dancetelevisionthree.m3u8
-#EXTINF:-1 tvg-id=" tvg-name="" tvg-country="US" tvg-language="Spanish"  tvg-logo="https://i.imgur.com/pb0b7A4.png" group-title="General",Telemundo 40 Arizona (720p)
+#EXTINF:-1 tvg-id="KTAZ.us" tvg-name="Telemundo 40 [KTAZ] (Phoenix, AZ)" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/pb0b7A4.png" group-title="General",Telemundo 40 Arizona (720p)
 https://ktazlive-lh.akamaihd.net/i/KTAZ_LIVE1@183609/master.m3u8
```
Comment (adamo331) [inline channels/us.m3u:3308]: tested in vlc off

**Pull request merged**